packages/block-library/src/search/view.js

```diff
@@ -66,6 +81,41 @@ const { actions } = store(
 					actions.closeSearchInput();
 				}
 			},
+			*updateSearch() {
+				const { ref } = getElement();
+				const { value } = ref;
+
+				// Don't navigate if the search didn't really change.
+				if ( value === state.search ) {
```

Comment:
@r-chrzan I've removed the checks for `name === 's'` here because I didn't quite see their purpose. Were they required for anything?

Reply:
@michalczaplinski it was related to previous implementation where we've had query loop variation